sn_networking/src/driver.rs

```diff
@@ -608,6 +615,21 @@ impl SwarmDriver {
                         bootstrap_interval = new_interval;
                     }
                 }
+                _ = set_farthest_record_interval.tick() => {
+                    let closest_peers = self
+                    .swarm
+                    .behaviour_mut()
+                    .kademlia
+                    .get_closest_local_peers(&self.self_peer_id.into())
+                    .map(|peer| peer.into_preimage())
+                    .take(CLOSE_GROUP_SIZE)
```

Comment:
`is_in_close_range` will always return true when the input list is less than REPLICATION_RANGE (i.e. 7)
but here there is only CLOSE_GROUP_SIZE (5) peers to be passed down ?